src/patternfly/components/Nav/nav.scss

```diff
@@ -251,8 +251,13 @@
       max-width: 100%;
       overflow-x: auto;
       white-space: nowrap;
+      scroll-snap-type: x proximity;
       -webkit-overflow-scrolling: touch;
 
+      > .pf-c-nav__item {
+        scroll-snap-align: end;
```

Comment:
You can save some lines of CSS and reduce the specificity of this selector by adding the `scroll-snap-align: end;` declaration to this block - https://github.com/patternfly/patternfly/blob/9dfbd1a9e1ffa053067e0835814f0653503cf939/src/patternfly/components/Nav/nav.scss#L264-L266

Reply:
Fixed, thanks!